api/server/index.js

```diff
@@ -41,6 +42,15 @@ config.validate(); // Validate the config
   if (process.env.FACEBOOK_CLIENT_ID && process.env.FACEBOOK_CLIENT_SECRET) {
     require('../strategies/facebookStrategy');
   }
+  if (process.env.OPENID_CLIENT_ID && process.env.OPENID_CLIENT_SECRET && process.env.OPENID_ISSUER && process.env.OPENID_SCOPE  && process.env.OPENID_SESSION_SECRET) {
+    app.use(session({
```

Comment:
I noticed that this seems to be using both JWT and sessions, but api's are authenticated by the JWT token. I had actually pulled out the use of sessions in favor JWT tokens as they are generally more secure. I'm not sure I'm following exactly how the session is being used... is that for validation from the 3rd party provider? If you could help clarify this, that would be great. thanks.